BUILD.md

```diff
@@ -132,3 +132,12 @@ see how to write a new Kanister Function.
 
 Don't forget to update the documentation at `docs/functions.rst` with
 configuration information and examples to show off your new function.
+
+## Build Kanister image
+
+Kanister build image [workflow](.github/workflows/kanister-image-build.yaml) is
+used to build and push new Kanister build image. It is on-demand workflow and
+you need to run it manually.
+This workflow expects `image tag` value as an input which gets used while
+building and pushing the image.
+To make the new tag persistent, you need to set it [here](https://github.com/kanisterio/kanister/blob/bf570c75cfaa28a0c9038c8c2a611e4a8a2e517a/Makefile#L61) and raise a PR.
```

Comment:
This is not very clear. We should say that the author updating the build image must raise a separate PR to update this value for it to be used in the build process.
Also, I would recommend changing the link to point to `master`. This `blob/bf570c75cfaa28a0c9038c8c2a611e4a8a2e517a` would become `blob/master`

Reply:
fixed